.gitignore

```diff
@@ -0,0 +1,13 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
```

Comment:
Currently do .gitignore using PyCharm which handles what files get pushed and what get ignored. The token file LECTURFIER.json is a folder up for an extra layer of security, so it doesn't get committed/pushed even if there's a mistake.